index.js

```diff
@@ -298,22 +298,77 @@ WBuf.prototype.writeUInt32BE = function writeUInt32BE(v) {
   }
 };
 
-WBuf.prototype.writeUInt16LE = function writeUInt16LE(num) {
```

Comment:
Shouldn't had `writeUInt16BE` below validate the `avail` value?

Reply:
I am not sure I can follow. I do not see any validation?

Reply:
It calls `this.writeUInt16BE`, not `this.last.writeUint16BE`, right?

Reply:
`WBuf.prototype.writeUInt16LE` used to call `this.writeUInt16BE` and that called `this.last.writeUInt16BE`. 

So I guess what you mean is that `WBuf.prototype.writeUInt16LE` does not have to be changed?

If not: can you please elaborate a bit more?

Reply:
Yeah, I mean that it probably should not be changed indeed.

Elaborated version: `WBuf#writeUInt16BE` performs length checks and does writes according to the available amount of bytes in the `last` buffer. Although, there're `noAssert = true` in `writeUInt16BE` - it should never cause any problems, since all writes are checked manually anyway. I understand that it gets deprecated, and I'm totally fine with removing them from the code due to this.

`WBuf#writeUInt16LE`, however only re-orders the bytes and forwards them to the aforementioned `WBuf#writeUInt16BE`. So even if we'd like to change one, there should not be a reason to change the second one, because they're practically running the same code.

Does it makes sense?

Reply:
Thanks for elaborating. And yes, I do see that it only reorders the numbers. I personally would still argue it makes sense to switch to the other version because the re-ordering is extra overhead that is unnecessary. Nevertheless: I changed it to the old code (and only renamed the variable for consistency).

Reply:
I don't think that the code was changed to what it was before :wink:

Reply:
You mean the renaming of `num` to `v`?

Reply:
No, I mean changing `writeUInt24LE` to what it was before.

Reply:
Addressed.

Reply:
Thanks! Would you care to  rename `v` back to `num` to minimize changed lines?

Reply:
Addressed.